presto-spi/src/main/java/com/facebook/presto/spi/type/TypeSignature.java

```diff
@@ -17,6 +17,7 @@
 import com.fasterxml.jackson.annotation.JsonValue;
 
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
Can you put up a separate PR for this commit and have @martint review? Or if it's already reviewed, we can push it now. It is unrelated to the Raptor changes and I don't have context to review it